server/@types/express/index.d.ts

```diff
@@ -26,6 +26,7 @@ import type {
   WorkInterestRolesForm,
   WorkInterestTypesForm,
 } from 'inductionForms'
+import type { InductionDto } from 'inductionDto'
```

Comment:
This type is used on `inductionDto` session property and VSCode needed this import to resolve it.